src/main/scala/scala/tools/refactoring/implementations/InlineLocal.scala

```diff
@@ -26,10 +26,12 @@ abstract class InlineLocal extends MultiStageRefactoring with ParameterlessRefac
       case None => s.findSelectedOfType[ValDef]
     }
 
+    def isInliningAllowed(sym: Symbol) =
+      (sym.isPrivate || sym.isLocal) && !sym.isMutable && !sym.isValueParameter
+
     selectedValue match {
-      case Some(t) if (t.symbol.isPrivate || t.symbol.isLocal)
-        && !t.symbol.isMutable && !t.symbol.isValueParameter
-        && t.symbol.enclMethod != NoSymbol => Right(t)
```

Comment:
yes, it was limited to method bodies but I don't understand why. Also, the condition checks for `sym.isPrivate` which can never be true for local values. This makes the code even more confusing.